pinot-core/src/main/java/org/apache/pinot/core/transport/QueryResponse.java

```diff
@@ -62,12 +62,28 @@ Map<ServerRoutingInstance, ServerResponse> getFinalResponses()
    */
   String getServerStats();
 
+  /**
+   * Returns the time taken for the server to respond to the query.
+   * @param serverRoutingInstance
+   */
+  int getServerResponseDelayMs(ServerRoutingInstance serverRoutingInstance);
```

Comment:
(nit) suggest keeping latency related metrics under same data type.

For example, like `timeOut` is tracked as long and `serverResponseDelay` (server latency including the transport time) is int

Reply:
Changed to long.
I had kept is as int because `ServerResponse.getResponseDelayMs` (from where this value is fetched) returns int.